src/views/ContinuationIn/ContinuationInAuthorization.vue

```diff
@@ -101,30 +144,46 @@ import { Component, Mixins, Watch } from 'vue-property-decorator'
 import { Getter, Action } from 'pinia-class'
 import { useStore } from '@/store/store'
 import { CommonMixin, NameRequestMixin } from '@/mixins'
+import { ExistingBusinessInfoIF } from '@/interfaces'
 import { ContactPointIF } from '@bcrs-shared-components/interfaces'
+import { ExpandableHelp } from '@bcrs-shared-components/expandable-help'
+import { JurisdictionLocation } from '@bcrs-shared-components/enums/'
 import AuthorizationProof from '@/components/ContinuationIn/AuthorizationProof.vue'
 import BusinessContactInfo from '@/components/common/BusinessContactInfo.vue'
 import ExtraproRegistration from '@/components/ContinuationIn/ExtraproRegistration.vue'
 import ManualBusinessInfo from '@/components/ContinuationIn/ManualBusinessInfo.vue'
 import NameRequestInfo from '@/components/common/NameRequestInfo.vue'
 import NameTranslations from '@/components/common/NameTranslations.vue'
+import UnlimitedLiabilityCorporationInformation
+  from '@/components/ContinuationIn/UnlimitedLiabilityCorporationInformation.vue'
+import UnlimitedLiabilityCorporationHelp from '@/components/ContinuationIn/UnlimitedLiabilityCorporationHelp.vue'
 
 @Component({
   components: {
     AuthorizationProof,
     BusinessContactInfo,
+    ExpandableHelp,
     ExtraproRegistration,
     ManualBusinessInfo,
     NameRequestInfo,
-    NameTranslations
+    NameTranslations,
+    UnlimitedLiabilityCorporationHelp,
+    UnlimitedLiabilityCorporationInformation
   }
 })
 export default class ContinuationInAuthorization extends Mixins(CommonMixin, NameRequestMixin) {
+  // Refs
+  $refs!: {
+    ulcInformation: UnlimitedLiabilityCorporationInformation
+  }
```

Comment:
Need this so we can reach into this child component and call one of its methods.